### PR TITLE
[CM] Display menu

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

It displays the menu in the CM

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js [hunk 1]
```diff
@@ -4,54 +4,106 @@
...
       title: {
-        id: getTrad('components.LeftMenu.single-types.'),
+        id: getTrad('components.LeftMenu.single-types'),
+        defaultMessage:
+          '{number, plural, =0 {Single Types} one {Single Type } other {Single Types}}',
+        values: { number: singleTypeLinks.length },
       },
       searchable: true,
-      links: singleTypeLinks,
+      links: sortBy(matchByTitle(singleTypeLinks, search), 'title'),
```
Comment (bulby97) [inline packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js:56]: For the CTB I kept the old search behaviour: Sections with no results are hidden.
Which behaviour should we enforce ?

**Review by bulby97 — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js (hunk 1 above)
Comment (soupette) [reply packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js:56]: Honestly I don't know

File: packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js [hunk 2]
```diff
@@ -1,64 +1,66 @@
...
-import { useSelector, shallowEqual } from 'react-redux';
-import axios from 'axios';
-import { axiosInstance } from '../../../core/utils';
-import { getRequestUrl, mergeMetasWithSchema } from '../../utils';
-import { makeSelectModelAndComponentSchemas } from '../App/selectors';
-import permissions from '../../../permissions';
-import crudReducer, { crudInitialState } from '../../sharedReducers/crudReducer/reducer';
-import { getData, getDataSucceeded } from '../../sharedReducers/crudReducer/actions';
-import EditSettingsView from '../EditSettingsView';
+// import React, { memo, useEffect, useMemo, useReducer } from 'react';
```
Comment (mfrachet) [inline packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js:1]: Why is this one commented? Should we remove it or uncomment?

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js [hunk 3]
```diff
@@ -295,14 +295,14 @@ describe('Admin | containers | ListView', () => {
...
-    const { container } = render(App);
+    const { container, getByText } = render(App);
     await waitFor(() => {
       screen.getByText('http:://strapi.io');
     });
 
     fireEvent.click(container.querySelector('#delete-1'));
-    await waitFor(() => {
-      expect(screen.getByText('Are you sure you want to delete this?')).toBeInTheDocument();
+    await waitFor(async () => {
```
Comment (mfrachet) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js:304]: you don't need the async, you need to remove the `{}` around the `expect()`

**Review by mfrachet — COMMENTED** [review]
Small comments, but looks good overall 👍🏻

File: packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js (hunk 2 above)
Comment (soupette) [reply packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js:1]: Right now I have commented all the unmigrated views. I'd rather keep it like this for the moment.

**Pull request merged**